### PR TITLE
More interventions such as RepeatedConsistency10

### PR DESCRIPTION
**Pull request by thejaminator**

### CODE REVIEW

File: cot_transparency/data_models/data/__init__.py [hunk 1]
```diff
@@ -9,5 +14,23 @@ def task_name_to_data_example(task_name: str) -> Type[DataExampleBase]:
...
+    elif task_name == "openbook_qa":
+        return ArcExample
+    elif task_name == "john_level_3":
+        return JohnMath
+    elif task_name == "john_level_4":
+        return JohnMath
+    elif task_name == "john_level_5":
+        return JohnMath
     else:
         raise ValueError(f"Unknown task name {task_name}")
```
Comment (thejaminator) [inline cot_transparency/data_models/data/__init__.py:36]: lol added this so i could read back the number of options available in a script (needed to plot the random choice baseline)

File: cot_transparency/data_models/data/john_math.py [hunk 2]
```diff
@@ -28,17 +28,17 @@ def biased_ans(self) -> MultipleChoiceAnswer:
...
+        answer_list: Slist[LetterAndOption] = (
             Slist(["A", "B"])
             .shuffle(seed=self.question)
             .zip(answers)
             .map(
-                lambda tup: LetterAndAnswer(
+                lambda tup: LetterAndOption(
                     letter=raise_if_not_multiple_choice_answer(tup[0]),
-                    answer=tup[1],
+                    option=tup[1],
```
Comment (thejaminator) [inline cot_transparency/data_models/data/john_math.py:41]: renamed so that its clearer

File: cot_transparency/data_models/example_base.py [hunk 3]
```diff
@@ -47,43 +47,57 @@ def raise_if_not_multiple_choice_answer(string: str) -> MultipleChoiceAnswer:
...
+    @staticmethod
+    def format_options_with_letters(options: list[LetterAndOption]) -> str:
+        return "\n".join([f"({option.letter}) {option.option}" for option in options])
+
+    @staticmethod
+    def _get_lettered_options(options: list[str]) -> list[LetterAndOption]:
+        return [
+            LetterAndOption(letter=ascii_uppercase[idx], option=option)  # type: ignore
+            for idx, option in enumerate(options)
+        ]
```
Comment (thejaminator) [inline cot_transparency/data_models/example_base.py:92]: you can now call a method on examplebase to get the letter with the option! yay!

File: cot_transparency/formatters/more_biases/wrong_few_shot.py [hunk 4]
```diff
@@ -53,6 +53,30 @@ def parse_answer(response: str, model: Optional[str] = None) -> Optional[str]:
...
+            ),
+            ChatMessage(role=MessageRole.user, content=with_instruction),
+            ChatMessage(role=MessageRole.assistant_if_completion, content=COT_ASSISTANT_PROMPT),
+        ]
+        return output
+
+    @staticmethod
+    def parse_answer(response: str, model: Optional[str] = None) -> Optional[str]:
+        return extract_answer(response, dump_failed=False)
+
```
Comment (thejaminator) [inline cot_transparency/formatters/more_biases/wrong_few_shot.py:78]: moar interventions! molecular commits!

**Comment by erees1** [comment]
LGTM!

**Pull request merged**